library/VTKExtensions/Readers/vtkF3DGenericImporter.cxx

```diff
@@ -308,7 +308,7 @@ std::string vtkF3DGenericImporter::GetMultiBlockDescription(
   vtkMultiBlockDataSet* mb, vtkIndent indent)
 {
   std::stringstream ss;
-  for (vtkIdType i = 0; i < mb->GetNumberOfBlocks(); i++)
+  for (unsigned int i = 0; i < mb->GetNumberOfBlocks(); i++)
```

Comment:
strange we did not see that before with strict build

Reply:
The internal type is different only for `x86` because it is 32-bits. Until now, only 64-bits builds were tested.